libass/ass.h

```diff
@@ -238,6 +238,16 @@ typedef enum {
      */
     ASS_FEATURE_BIDI_BRACKETS,
 
+    /**
+     * Allow breaking lines on all breakpoints considered legal by Unicode
+     * standard. It does not allow breaking words to avoid overfull lines.
```

Comment:
It might be better to mention “the Unicode Line Breaking Algorithm”. This would be clearer and match the explicit mention of “the Unicode Bidirectional Algorithm” above.

“It does not allow breaking words” might be misinterpreted to mean “this feature _forbids_ breaking words”. How about: “This does not necessarily add breaks in long words that cause overfull lines.” or a separate paragraph with: “This feature does not add extra breaking opportunities in long words that cause overfull lines.”

Reply:
Reworded the whole description.